akka-interop/src/main/scala/zio/actors/akka/AkkaTypedActorRefLocal.scala

```diff
@@ -0,0 +1,50 @@
+package zio.actors.akka
+
+import akka.actor.typed
+import akka.actor.typed.Scheduler
+import akka.actor.typed.scaladsl.AskPattern._
+import akka.util.Timeout
+import zio.{ Task, UIO, ZIO }
+
+/**
+ *
+ * Reference to proxy actor that contains an akka typed actor ref
+ *
+ * @tparam F wrapper type constructing DSL
+ */
+private[actors] final class AkkaTypedActorRefLocal[-F[+_]](actorName: String, akkaActor: typed.ActorRef[F[_]])
+    extends Serializable {
+
+  /**
+   *
+   * Send message to an actor as fire-and-forget
+   *
+   * @param fa message
+   * @return lifted unit
+   */
+  def !(fa: F[_]): Task[Unit] =
+    for {
+      _ <- UIO(akkaActor ! fa)
+    } yield ()
```

Comment:
Here also can be simplified to just `UIO(akkaActor ! fa)` without `for` comprehension